conf/ephys_upload_job_configs.yml

```diff
@@ -20,8 +30,9 @@ compress_data_job:
     chunk_size:
     disable_tqdm:
 upload_data_job:
-  dryrun: false # Set this as false to perform the actual upload
-  upload_to_s3: true # Set as false to skip uploading to s3
-  s3_dest:  # Set to something like s3://BUCKET/PATH
-  upload_to_gcp: true # Set as false to skip uploading to s3
-  gcp_dest:  # Set to something like gs://BUCKET/PATH
+  dryrun: false # Set this to true to perform a dryrun of the upload
+register_on_codeocean_job:
+  api_token: # Defaults to ENV Variable CODEOCEAN_API_TOKEN
+  tags: ['ecephys']
```

Comment:
can we add a `raw` tag here?

Reply:
Done. In the future, we should consider extracting out aind dependent defaults from the config template to make everything more open source. Maybe we can add a config file creator script in the future?

Reply:
Yeah, that's a fine idea.